src/ServiceFabric/ServiceFabric.Management.Sdk/README.md

```diff
@@ -20,7 +20,7 @@ payload-flattening-threshold: 2
 use-extension:
```

Comment:
READ.md is not in correct format. Please refer to https://github.com/Azure/azure-powershell/blob/main/src/KeyVault/KeyVault.Sdk/README.md to correct the file

Reply:
Made some changes but these files readme looks similar to the one mentioned above. Let me know if  I am missing something